src/Material.cpp

```diff
@@ -563,7 +563,7 @@ void Material::setSigmaT(double* xs, int num_groups) {
 
     /* If the cross-section is near zero (e.g., within (-1E-10, 1E-10)) */
     if (fabs(xs[i]) < ZERO_SIGMA_T) {
-      log_printf(INFO, "Overriding zero cross-section in "
+      log_printf(INFO, "Overriding zero tot. MGXS in "
```

Comment:
Yeah so the reason is simply that the message length with "total" exceeds 80 characters and is split across two lines. This can clutter an output log when you have lots (100s-1000s) of `Materials` with lots (10s) of energy groups. If you think "tot." is confusing then I'll switch it to "total" though.